spec/lib/breakers/statsd_plugin_spec.rb

```diff
@@ -53,6 +53,22 @@
 
         request.url = URI(test_host + '/v0.0/Providers(1234567890)/bar')
         expect(subject.get_tags(request)).to include('endpoint:/v0.0/xxx/bar')
+
+        request.url = URI(test_host + '/api/v1/users/b3363659ac50d661149470')
+        expect(subject.get_tags(request)).to include('endpoint:/api/v1/users/xxx')
+
+        request.url = URI(test_host + '/api/v1/users/00u2sgjcthlgio12o297')
+        expect(subject.get_tags(request)).to include('endpoint:/api/v1/users/xxx')
+
+        request.url = URI(test_host + '/api/v1/users/00u2i1p2u2m3l7FYb712/grants')
+        expect(subject.get_tags(request)).to include('endpoint:/api/v1/users/xxx/grants')
+      end
+    end
+
+    context 'request without an id' do
+      it 'doesnt replace anything' do
+        request.url = URI(test_host + '/foo/bar')
+        expect(subject.get_tags(request)).to include('endpoint:/foo/bar')
```

Comment:
Added a "negative case" spec. Not specific to this PR, but I wanted to be sure that I wasnt blowing away "normal" url paths that _didnt_ contain dynamic ID's.